hpat/tests/test_series.py

```diff
@@ -902,6 +902,14 @@ def test_impl(S):
         S = pd.Series([np.nan, 2., 3.])
         self.assertEqual(hpat_func(S), test_impl(S))
 
+    def test_series_min(self):
+        def test_impl(S):
+            return S.min()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([5, 6, 13])
```

Comment:
May be, it's better to check not only Integer values? What if we add some nan elements?

Reply:
and float, and negatives and `None`